### ISSUE TITLE
Version 5.1.5 crashes on screen record

### ISSUE THREAD

**Issue by GF-Ribeiro** [RenderHeads/UnityPlugin-AVProMovieCapture#285]
**Describe the bug**
Unity Crashes when I try to record a screen video. I believe this is a problem with Version 5.1.5.
I had the trial version 5.1.4 working just fine, then I bought the asset, and when I updated to the complete version it started to happen. I have tried on an empty project and on newer versions of Unity and got the same result: unity crashes when it tries to record.
I suspect version 5.1.4 will solve my problem, as I was experiencing no problems with the trial. If you can make it available, it would be much appreciated. I am in a rush to finish a project and a quick support will save my life.

Thank you!
Best,
Gabriel R.

**Your Setup (please complete the following information):**
 - Unity version: 2021.3.5f1 / 2021.3.25f1 / 2022.2.19f1
 - AVPro Movie Capture version: 5.1.5
 - Operating system version: Windows 11
 - Capture component used: Capture from screen 
 - Capture settings (resolution, frame-rate, codec): Standart with Audio set to none

**To Reproduce**
1. Just create a new project on Unity 2021.3.5f1 or newer
2. Try to record screen

<details>
<summary>Logs</summary>

```
0x000001e1dbb33c76 (AVProMovieCapture) AVPMC_CreateRecorderVideo
0x000001e262876f90 (Mono JIT Code) (wrapper managed-to-native) RenderHeads.Media.AVProMovieCapture.NativePlugin:CreateRecorderVideo (string,uint,uint,single,int,bool,bool,int,RenderHeads.Media.AVProMovieCapture.AudioCaptureSource,int,int,int,int,bool,RenderHeads.Media.AVProMovieCapture.VideoEncoderHints)
0x000001e26286dffb (Mono JIT Code) RenderHeads.Media.AVProMovieCapture.CaptureBase:PrepareCapture () (at D:/Projetos/Pessoal/Test Video/Assets/Plugins/RenderHeads/AVProMovieCapture/Runtime/Scripts/Internal/CaptureBase.cs:2354)
0x000001e262855b53 (Mono JIT Code) RenderHeads.Media.AVProMovieCapture.CaptureFromScreen:PrepareCapture () (at D:/Projetos/Pessoal/Test Video/Assets/Plugins/RenderHeads/AVProMovieCapture/Runtime/Scripts/Components/CaptureFromScreen.cs:168)
0x000001e2628541bd (Mono JIT Code) RenderHeads.Media.AVProMovieCapture.CaptureBase:StartCapture () (at D:/Projetos/Pessoal/Test Video/Assets/Plugins/RenderHeads/AVProMovieCapture/Runtime/Scripts/Internal/CaptureBase.cs:2531)
0x000001e262853f03 (Mono JIT Code) RenderHeads.Media.AVProMovieCapture.CaptureGUI:StartCapture () (at D:/Projetos/Pessoal/Test Video/Assets/Plugins/RenderHeads/AVProMovieCapture/Runtime/Scripts/Components/CaptureGUI.cs:817)
0x000001e2612e8563 (Mono JIT Code) RenderHeads.Media.AVProMovieCapture.CaptureGUI:MyWindow (int) (at D:/Projetos/Pessoal/Test Video/Assets/Plugins/RenderHeads/AVProMovieCapture/Runtime/Scripts/Components/CaptureGUI.cs:553)
0x000001e2612e11bd (Mono JIT Code) UnityEngine.GUILayout/LayoutedWindow:DoWindow (int)
0x000001e2612dfa4b (Mono JIT Code) UnityEngine.GUI:CallWindowDelegate (UnityEngine.GUI/WindowFunction,int,int,UnityEngine.GUISkin,int,single,single,UnityEngine.GUIStyle)
0x000001e2612e0262 (Mono JIT Code) (wrapper runtime-invoke) <Module>:runtime_invoke_void_object_int_int_object_int_single_single_object (object,intptr,intptr,intptr)
0x00007ffb542be434 (mono-2.0-bdwgc) mono_jit_runtime_invoke (at C:/build/output/Unity-Technologies/mono/mono/mini/mini-runtime.c:3445)
0x00007ffb541feb44 (mono-2.0-bdwgc) do_runtime_invoke (at C:/build/output/Unity-Technologies/mono/mono/metadata/object.c:3066)
0x00007ffb541fecdc (mono-2.0-bdwgc) mono_runtime_invoke (at C:/build/output/Unity-Technologies/mono/mono/metadata/object.c:3113)
```
</details>

**Comment by Chris-RH** [comment]
Hi, 

Sorry about that. It just seems to be the Basic Edition that is affected.  We'll put up a new version after the weekend when we are back in the office.

**Comment by MorrisRH** [comment]
We've released version 5.1.6 which includes a fix for this issue.
